.client-env.example

```diff
@@ -0,0 +1,19 @@
+AUTH_KEYCLOAK_URL=https://<HOSTNAME>/auth
+AUTH_KEYCLOAK_REALM=metadata-editor
+AUTH_KEYCLOAK_CLIENT_ID=mde
+AUTH_KEYCLOAK_CLIENT_SECRET=
+# This is a route in the client
+AUTH_REDIRECT_URI=https://<HOSTNAME>/auth-callback
+
+# The Backend URL as accesible from the docker network
+BACKEND_URL=http://mde-backend:8080
+
+# The log level for the client
+CLIENT_LOG_LEVEL=debug
+
+# Allows self-signed certificates
+NODE_TLS_REJECT_UNAUTHORIZED=0
+
+#TODO: This should be moved to the docker setup
+
+AUTH_SECRET="Jne6EkxFlOI8GgONMQytHhhUqmHgHCq2" # Added by `npx auth`. Read more: https://cli.authjs.dev
```

Comment:
Not sure if you want to remove this value before merging.

Reply:
Removed it